test/interceptors/csrf-token.test.js

```diff
@@ -85,4 +85,18 @@ describe('CSRF token', () => {
         expect(axiosMock.defaults.headers.requesttoken).toBe('123')
         expect(response?.status).toBe(200)
     })
+
+    it('intercepts a cancellation error', () => {
+        const cancelError = {
+            code: 'ERR_CANCELED',
+            message: 'canceled',
+            name: 'CanceledError',
+            stack: '',
+        }
+        interceptor(cancelError).catch(error => {
```

Comment:
right now the error will be thrown/caught after completion of the test because the test is written with a synchronous closure but uses async (promise) code.

change the closure to an async one, wrap the interceptor call with try-catch and await the result like two tests above

Reply:
:+1: